### DISCUSSION TITLE
Notification from HttpTransferRequest crashes when tapped

### DISCUSSION THREAD

**Discussion by henda79** [shinyorg/shiny#1226] [Q&A]
I'm using the IHttpTransferDelegate which in turn uses INotificationManager to send a local notifications. However if the user taps the notification the app crashes as the main activity is created again.

This is on the android platform.

Is this a known issue or would you like a small repo and an issue filed ?

**Comment by aritchie** [comment]
You have access to the same issue repo and templates as everyone else.  Please use them

**Reply by henda79** [reply]
Just seems a bit pointless wasting time on creating a demo to file an issue if its already known. I've trawled through the resources here but can't find anything related so I'll assume its an unknown bug.

I'll create a demo and file an issue over the next day or so.
